kie-dmn-core/src/main/java/org/kie/dmn/core/compiler/DMNFEELHelper.java

```diff
@@ -44,6 +48,32 @@ private FEEL createFEELInstance() {
     public void onEvent(FEELEvent event) {
         feelEvents.add( event );
     }
+    
+    public static boolean valueMatchesInUnaryTests(List<UnaryTest> unaryTests, Object value, DMNContext dmnContext) {
+        EvaluationContextImpl ctx = new EvaluationContextImpl( null );
+        try {
+            ctx.enterFrame();
+            // need to set the values for in context variables...
+            for ( Map.Entry<String,Object> entry : dmnContext.getAll().entrySet() ) {
+                ctx.setValue( entry.getKey(), entry.getValue() );
+            }
+    
+            for ( UnaryTest t : unaryTests ) {
+                try {
+                    Boolean applyT = t.apply(ctx, value);
+                    if ( applyT ) {
+                        return true;
+                    }
+                } catch ( Throwable e ) {
+                    // if an unaryTest fail for any treason, is simply considered FALSE.
+                    logger.error("An error happended while evaluating a unary test", e);
```

Comment:
Sorry for changing my mind, but I think I agree with you. Lets downgrade this to a logger.warn() and change the message to: "A non-critical error happened while evaluating a unary test. Execution will continue.".